tests/sentry/integrations/github/test_integration.py

```diff
@@ -623,11 +622,9 @@ def test_get_trees_for_org(self):
                 {"full_name": "Test-Organization/bar", "default_branch": "main"},
                 {"full_name": "Test-Organization/baz", "default_branch": "master"},
             ]
-            assert cache.get("githubtrees:repo:Test-Organization/foo") == RepoTree(
-                Repo("Test-Organization/foo", "master"),
-                ["src/sentry/api/endpoints/auth_login.py"],
-            )
-
+            assert cache.get("github:repo:Test-Organization/foo:source-code") == [
+                "src/sentry/api/endpoints/auth_login.py"
```

Comment:
We do not cached anymore a `RepoTree` but a list of files.
<img width="405" alt="image" src="https://user-images.githubusercontent.com/44410/204311029-f63674a2-44b2-42d0-9348-dd24edb9e882.png">
vs
<img width="334" alt="image" src="https://user-images.githubusercontent.com/44410/204310942-1372450b-4960-436e-8797-f4beab1bb6e6.png">